### ISSUE TITLE
Windows release

### ISSUE THREAD

**Issue by sevenfm** [civctp2/civctp2#379]
Is there a possibility to make a windows release, for those who have no idea how to compile it?
I'm a long time fan of CtP2 and AE edition and wish to play this mod and maybe report bugs if found, but I doubt I could properly build it from repository.

**Comment by orlovskyjavaprofi** [comment]
As far as i know you need visual studio there once loaded sln project you can may try to compile.

**Comment by LynxAbraxas** [comment]
> Is there a possibility to make a windows release, for those who have no idea how to compile it?

Yes possibilities exist, but have not been realized so far (contributions welcome!).
AppVeyor CI could be used for Windows builds, see #6 and as reference see also: #348 and #294.

**Comment by LynxAbraxas** [comment]
@thistleknot did you succeed in the end on #348?

**Comment by thistleknot** [comment]
no, I kind of moved on from games these days.

**Comment by LynxAbraxas** [comment]
Seems GL-CI now offers shared runners for building on windows:
https://about.gitlab.com/blog/2020/01/21/windows-shared-runner-beta/
which already provide VS:
https://gitlab.com/gitlab-org/ci-cd/shared-runners/images/gcp/windows-containers/blob/main/cookbooks/preinstalled-software/README.md
So we would just need a contribution to 
https://github.com/civctp2/civctp2/blob/92cb24b7e773cce8987a6e5aa9307026a483efbd/.gitlab-ci.yml#L17-L23
to also build the current sources for CTP2, not sure though if that could be achieved with the current use of `dind`
